fetchai/communication.py

```diff
@@ -65,9 +66,16 @@ def _digest(self) -> bytes:
 
 
 def lookup_endpoint_for_agent(agent_address: str) -> str:
+    request_meta = {
+        "agent_address": agent_address,
+        "lookup_url": DEFAULT_ALMANAC_API_URL,
+    }
+    logger.debug("looking up endpoint for agent", extra=request_meta)
     r = requests.get(f"{DEFAULT_ALMANAC_API_URL}/agents/{agent_address}")
     r.raise_for_status()
 
+    request_meta["response_status"] = r.status_code
+    logger.info("Got response looking up agent endpoint")
```

Comment:
should this also have the request meta on it?

Reply:
Oooh good point!

Reply:
[44b3867](https://github.com/flockx-official/fetchai/pull/29/commits/44b3867843283014a7ab0c7bdff5f1d050d349ad)